app/src/modules/settings/routes/extensions/components/extension-item.vue

```diff
@@ -2,12 +2,14 @@
 import api from '@/api';
 import VChip from '@/components/v-chip.vue';
 import VProgressCircular from '@/components/v-progress-circular.vue';
-import type { ApiOutput } from '@directus/extensions';
+import type { ApiOutput, ExtensionType } from '@directus/extensions';
 import { computed, ref } from 'vue';
 import { useI18n } from 'vue-i18n';
-import { iconMap } from '../constants/icons';
+import { iconMap } from '../utils/icons';
 import ExtensionItemOptions from './extension-item-options.vue';
 
+const devMode = process.env.NODE_ENV === 'development';
```

Comment:
Interesting where does `import.meta.env.DEV` get set? I can see it being used but it's not obvious when/where it applies 🤔 I went for `process.env.NODE_ENV === 'development'` because we explicitly set it and use that in places like the vite config

Reply:
It's a specially exposed variable by Vite, see [import.meta.env](https://vitejs.dev/guide/env-and-mode#env-variables). As far as I understand it, designed exactly for such cases.
Feels a bit safer than relying directly on `process.env` which is theoretically unavailable in browser env.